pygfunction/gfunction.py

```diff
@@ -12,13 +12,18 @@
 from .heat_transfer import thermal_response_factors, finite_line_source
 from .networks import Network, network_thermal_resistance
 
+
 class gFunction(object):
-    def __init__(self, boreholes_or_network, alpha, time=None,
-                 method='similarities', boundary_condition=None, options=None):
```

Comment:
@MassimoCimmino You have method here as a string. Is `use_similarities` more than a True/False? I would understand having a `str` for the input if there's more than just two options or there will be.

Reply:
I expect there will be more than two options in the near future. We have some upcoming work that might be of interest to pygfunction.

Reply:
Great, then I'll need to revert back to using type `str` for the `method`.

One thing though, there is a `method` in `options={}` because you can specify `linear`, etc. for the `scipy.interp` stuff.

Reply:
The interpolation option could be replaced by `kind` to align with `scipy` terminology?

Reply:
Yes replacing `method` with `kind` for the `scipy.interp1d` option sounds good to me.